Makefile

```
@@ -83,6 +94,16 @@ docs: check-venv
 session-docx: check-venv
 	$(PYTHON) infrastructure/scripts/session_to_docx.py
 
+release: test security
```

checkmake (0.3.2)

[warning] Target body for "release" exceeds allowed length of 5 lines (8).

(maxbodylength)